users/README.md

```diff
@@ -1,27 +1,87 @@
 # Users
 
+```json
+{
+	"id": "string",
+	"username": "string",
+	"first_name": "string",
+	"last_name": "string",
+	"email": "string",
+	"phone": "number",
+	"yoe": "number",
+	"company": "string",
+	"designation": "string",
+	"img": "string",
+	"github_id": "string",
+	"github_display_name": "string",
+	"linkedin_id": "string",
+	"twitter_id": "string",
+	"instagram_id": "string",
+	"skills": [],
+	"joined_discord": "string",
+	"website": "string",
+	"isMember": "boolean",
+	"userType": "string",
```

Comment:
Also about this

Reply:
This was already in the document, so will address this in later PR if needed.